assignment4/index.js

```diff
@@ -0,0 +1,138 @@
+import todolists from './Todolist.js';
+
+//todoList의 개수를 세는 함수
+const countTodolist = () => {
+    let sum = 0;
+    todolists.forEach((todoL) => {
+        sum += todoL.todo.length;
+    });
```

Comment:
오우 깔끔하네요!!!
구조분해할당을 이용한다면!!
```suggestion
todolists.forEach(({todo}) => {
        sum += todo.length;
    });
```
쪼꼼 더 깔끔해질 수 있어요 ㅎㅎ

Reply:
[Array.reduce](https://miiingo.tistory.com/365)사용해서 하는게 더 깔끔하죠잉!